justfile

```diff
@@ -440,7 +451,62 @@ _linkerd-viz-uninit:
 
 
 # TODO linkerd-jaeger-install
-# TODO linkerd-multicluster-install
+
+##
+## linkerd multicluster
+## 
+
+mc-gateway-image := "gcr.io/google_containers/pause:3.2"
+
+linkerd-mc *flags: _k3d-init
+    {{ _linkerd }} mc {{ flags }}
+
+linkerd-mc-install: _linkerd-init linkerd-mc-load && _linkerd-mc-ready
+    {{ _linkerd }} mc install \
+            --set='linkerdVersion={{ linkerd-tag }}' \
+        | {{ _kubectl }} apply -f -
+
+# Wait for all test namespaces to be removed before uninstalling linkerd-multicluster.
+linkerd-mc-uninstall:
+    {{ _linkerd }} mc  uninstall \
+        | {{ _kubectl }} delete -f -
+
+linkerd-mc-build:
+	TAG={{ linkerd-tag }} bin/docker-build-controller
+
+linkerd-mc-load: _linkerd-mc-images _k3d-init
+	{{ _k3d-load }} \
+		{{ controller-image }}:{{ linkerd-tag }} \
+		{{mc-gateway-image }}
```

Comment:
With the extension of `linkerd-mc-buidl`, these are all going to use the `k3d-name` cluster and not the source or target cluster, so I'm not sure they're useful in their current form.